uwblueprint/sistering
# Lena/admin schedule state management

## Implementation description
* Had a lot of issues with the types for the `ADMIN_SCHEDULE_TABLE_DATA_QUERY` so I made ones specific to the fetch we were doing on the admin schedule page
* changed `currentlyEditingSignups` to be a shift to make handling the save easier
* mutation call for saving changes to signups


## Steps to test
1. Login as an admin and go to `http://localhost:3000/admin/schedule/posting/1`
2. Select a shift in the side panel that has signups
3. Checkmark/uncheckmark some signups and save
4. Check that the save button loads and then changes back to "Edit"
5. Refresh the page to check that your changes persist


## What should reviewers focus on?
* The app functionality still makes sense

You should be able to look at each commit individually to make reviewing easier. You can skip the first two though since they've already been reviewed (I made the mistake of working on the same branch). 

## Checklist
